years.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if (grep -L -r 2011-`date +%Y` --exclude-dir ".git" --exclude ".*" --exclude-dir "est" --exclude "*.yml" --exclude "*.md" --exclude-dir "target" --exclude-dir "it" . | egrep -v "(qulice-pmd/src/test/resources|qulice-checkstyle/src/test/resources|qulice-xml/src/test/resources|src/site/resources/CNAME|qulice-codenarc/src/main/resources/com/qulice/codenarc/rules.txt|qulice-gradle-plugin/src/main/resources/META-INF/gradle-plugins/qulice-plugin.properties|years.sh)"); then
+    echo "Files above have wrong years in copyrights"
+    exit 1
+else
```

Comment:
@yegor256 Sorry, I missed this rule a lot, it is quite obvious but I'm not used to it yet.
I crated an issue to add this check to qulice, should be doable with checkstyle for both java na sh - https://github.com/teamed/qulice/issues/628